mesh/ncmesh.hpp

```diff
@@ -1011,6 +1019,8 @@ class NCMesh
 
    static GeomInfo GI[Geometry::NumGeom];
 
+   Array<int> vertex_to_knot;
```

Comment:
The exact meaning and usage of the variable  is not completely clear to me.
The knot here refers to a single knot, right? Not the entire knotvector?
Why do the parents need to be included? The variable seems to do a bit more then advertised. This might be necessary, or desirable, but perhaps the name should be changed, or the parents should be dropped...

These concerns might be completely rooted in my misunderstanding, so please educate me.